node/node.go

```diff
@@ -910,14 +910,14 @@ func (node *AlgorandFullNode) SetCatchpointCatchupMode(catchpointCatchupMode boo
 		node.log.Infof("Indexer is not available - %v", err)
 	}
 
-	node.cancelCtx()
+	// Set up a context we can use to cancel goroutines on Stop()
```

Comment:
Wouldn't you still want to keep a `node.cancelCtx()` before re-initializing here since it's not explicitly called?

Reply:
This should not be required : the existing context at that time is only used by the catchpoint service itself. The catchpoint service is going to use the return value from this function to update it's context, so it won't have any affect.